src/tests/audio/tasks/forced_alignment_test.py

```diff
@@ -0,0 +1,215 @@
+"""Tests for forced alignment functions."""
+
+import os
+from typing import Optional, TypedDict
+
+import numpy as np
+import pandas as pd
+import pytest
+import torch
+from transformers import Wav2Vec2ForCTC, Wav2Vec2Processor
+
+from senselab.audio.data_structures.audio import Audio
+from senselab.audio.tasks.forced_alignment.data_structures import (
+    Point,
+    SingleSegment,
+)
+from senselab.audio.tasks.forced_alignment.forced_alignment import (
+    _align_segments,
+    _align_transcription,
+    _can_align_segment,
+    _get_prediction_matrix,
+    _interpolate_nans,
+    _merge_repeats,
+    _preprocess_segments,
+    align_transcriptions,
+)
+from senselab.utils.data_structures.device import DeviceType
+from senselab.utils.data_structures.language import Language
+from senselab.utils.data_structures.script_line import ScriptLine
+
+MONO_AUDIO_PATH = "src/tests/data_for_testing/audio_48khz_mono_16bits.wav"
+
+
+@pytest.fixture
+def mono_audio_sample() -> Audio:
+    """Fixture for sample mono audio."""
+    return Audio.from_filepath(MONO_AUDIO_PATH)
+
+
+class SingleCharSegment(TypedDict):
```

Comment:
how is this useful?

Reply:
Haha, great question--it's not. Removed.